forge/db/controllers/Invitation.js

```diff
@@ -105,7 +105,15 @@ module.exports = {
         await invitation.destroy()
         await app.notifications.remove(invitedUser, notificationReference)
 
+        // detail in audit log
         app.auditLog.Team.team.user.invite.accepted(user, null, invitation.team, invitedUser, role)
+        // send invitor a notification
+        app.notifications.send(invitation.invitor, 'team-invite-accepted-invitor', {
+            team: invitation.team,
```

Comment:
Need to be careful to not inadvertently expose raw object IDs here - the notification API won't know how to sanitize.

Id suggest storing the minimum information needed - explicitly pull out just the names, rather than referring the whole object.